scripts/tests/test_zkai_tablero_hybrid_zkml_boundary_gate.py

```
@@ -0,0 +1,222 @@
+        for row in payload["boundary_examples"]:
+            self.assertEqual(set(row), expected)
+            self.assertEqual(row["statement_commitment"], gate.statement_commitment(row))
+            for field in ("model_binding", "input_binding", "output_binding", "proof_object_binding"):
+                self.assertEqual(set(row[field]), set(gate.BINDING_OBJECT_FIELDS))
+                self.assertTrue(row[field]["commitment"].startswith("blake2b-256:"))
+
+    def test_rejects_compact_statement_as_native(self) -> None:
+        payload = gate.build_payload()
+        gate.promote_compact_statement_as_native(payload)
+        payload["payload_commitment"] = gate.payload_commitment(payload)
+        with self.assertRaisesRegex(gate.TableroHybridBoundaryError, "native proof equivalence overclaim"):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)